spec.md

```diff
@@ -184,16 +184,16 @@ as a string.
   -  String encoding: Base64 encoding per [RFC4648](https://tools.ietf.org/html/rfc4648).  
 - `Map` - `String`-indexed dictionary of `Any`-typed values.
   - String encoding: JSON Object per [RFC 7159, Section 4](https://tools.ietf.org/html/rfc7159#section-4)
-- `Any` - Either a `Binary`, `Integer`, `Map` or `String`.
 - `URI-reference` - Uniform resource identifier reference.
    - String encoding: `URI-reference` as defined
   in [RFC 3986 Section 4.1](https://tools.ietf.org/html/rfc3986#section-4.1).
 - `Timestamp` - Date and time expression using the Gregorian Calendar. 
    - String encoding: [RFC 3339](https://tools.ietf.org/html/rfc3339).
+- `Any` - A variant type that can take the shape of either an `Integer`,
+  `String`, `Binary`, `Map`, `URI-reference` or `Timestamp`.
 
-The `Any` type is a variant type that can take the shape of either a `Binary`,
-`Integer`, `Map` or `String`. The type system is intentionally abstract, and
-therefore it is left to implementations how to represent the variant type.
+The type system is intentionally abstract, and therefore it is left to
```

Comment:
I'll let @clemensv comment on the wording change, but I think these 2 lines need to be indented and the blank line removed so they appear under the "Any" section.

Reply:
I left it on a separate line after that list on purpose, as it was like that before and is a rather generic remark on the type-system. But I can also attach it just to that last item on the `Any` type. No strong preference.

Reply:
I think the diff here is that since you moved Any to the end of the list (which I like since Any is kind of a "catch-all/everything-else" type of option), the split that was there now looks odd since both sections are "Any" specific and so grouping them together looks nicer to me.

Reply:
Ok, I've pulled the both sections together and force-pushed. While at it, I've also fixed an unrelated typo.